contrib/completion/zsh/_docker

```diff
@@ -1845,6 +1845,7 @@ __docker_service_subcommand() {
             _arguments $(__docker_arguments) \
                 $opts_help \
                 "($help)*"{-f=,--filter=}"[Filter output based on conditions provided]:filter:->filter-options" \
+                "($help)--format=[Pretty-print services using a Go template]:template: " \
```

Comment:
Sorry, format was not the right word given that's the flag here... but looks like flags with a single representation use a different syntax than ones with multiple representations (e.g. `-q` and `--quiet` vs simply `--format`).